httpclient/client.go

```diff
@@ -0,0 +1,42 @@
+package httpclient
+
+import (
+	"fmt"
+	"net/http"
+
+	transactionidutils "github.com/Financial-Times/transactionid-utils-go"
+)
+
+type httpClient interface {
+	Do(req *http.Request) (*http.Response, error)
+}
+
+type ServiceClient struct {
+	Client      httpClient
+	ServiceCode string
+	Version     string
+	Runbook     string
+}
+
```

Comment:
All comments in the method's implementation could be removed and we could add a doc comment for what the method does.